src/groupeddataframe/complextransforms.jl

```diff
@@ -110,7 +118,20 @@ function _combine_rows_with_first!(first::Union{NamedTuple, DataFrameRow},
     # Handle first group
     j = fill_row!(first, outcols, rowstart, colstart, colnames)
     @assert j === nothing # eltype is guaranteed to match
-    # Handle remaining groups
+    # Handle remaining groups (eliminate the argument type-diversity of `first`)
```

Comment:
lines https://github.com/JuliaData/DataFrames.jl/pull/2563/files#diff-fe9f0d31012cded8c66d568e600c72b8b7a55ad5e4826bf645d885c7aed0a76eR111-R113 above can be removed as they are not used in this method now.